packages/hathor-rpc-handler/src/rpcMethods/sendNanoContractTx.ts

```diff
@@ -20,7 +21,20 @@ import {
   RpcResponse,
   SendNanoContractTxLoadingFinishedTrigger,
 } from '../types';
-import { PromptRejectedError, SendNanoContractTxError } from '../errors';
+import { PromptRejectedError, SendNanoContractTxError, InvalidParamsError } from '../errors';
+import { NanoContractAction } from '@hathor/wallet-lib/lib/nano_contracts/types';
```

Comment:
Maybe we should expose these types in the lib index instead of importing from the internal path.

Reply:
I agree, I will create an issue in the wallet-lib